SteamKit2/SteamKit2/Steam/CDN/DepotChunk.cs

```diff
@@ -53,16 +53,29 @@ public static int Process( DepotManifest.ChunkData info, ReadOnlySpan<byte> data
             try
             {
                 var written = aes.DecryptCbc( data[ iv.Length.. ], iv, buffer, PaddingMode.PKCS7 );
-                var decryptedStream = new MemoryStream( buffer, 0, written );
 
-                if ( buffer.Length > 1 && buffer[ 0 ] == 'V' && buffer[ 1 ] == 'Z' )
+                if ( buffer.Length < 16 )
```

Comment:
Steam client checks for like 20 bytes for pkzip, and 22 bytes for vzip, I'm just being safe and checking for a smaller value.